### DISCUSSION TITLE
Link problem using walletConnect2

### DISCUSSION THREAD

**Discussion by Amazing123511** [WalletConnect/walletconnect-monorepo#2336] [Developer Support]
I have been reporting this link error, how to solve it? Is there a problem with the projectId?   
index.js:7     POST https://verify.walletconnect.com/attestation 422

**Comment by finessevanes** [comment]
Hey - please follow the support [here](https://github.com/orgs/WalletConnect/discussions/2329). Closing since this is a duplicate discussion.
